### PR TITLE
Fix some errors in the English of the SES section (LGHP)

### PR DESCRIPTION
**Pull request by lproven**
Fix the phrasing of the English, and use the existing entity when introducing SES.

### CODE REVIEW

File: xml/admin_integration.xml [hunk 1]
```diff
@@ -30,14 +30,14 @@
...
-     &productname; can access the SES cluster.
+     The &productname; cluster can access the SES cluster.
     </para>
    </listitem>
    <listitem>
     <para>
-     &productname; can communicate with the SES nodes&mdash;master, monitoring
-     nodes, OSD nodes and the meta data server in case you need a shared
-     file system. For more details regarding SES refer to
+     That &productname; can communicate with the SES nodes&mdash;master,
```
Comment (cwickert) [inline xml/admin_integration.xml]: "That" seems wrong here.

File: xml/admin_integration.xml [hunk 2]
```diff
@@ -30,14 +30,14 @@
...
+     monitoring nodes, OSD nodes and the metadata server, in case you need a
+     shared file system. For more details regarding SES refer to
      <link xlink:href="https://www.suse.com/documentation/ses-4/book_storage_admin/data/book_storage_admin.html">SES
```
Comment (cwickert) [inline xml/admin_integration.xml]: I think we need an article here "the SES documentation", not just "SES documentation". Please correct me if I'm wrong.

Shouldn't we use an entity for SES anyway? Just saying, that might be better taken care off in a different PR.

File: xml/admin_integration.xml (hunk 1 above)
Comment (lproven) [reply xml/admin_integration.xml]: No, it was intentional, but I have revised it to be consistent with the previous line.

File: xml/admin_integration.xml (hunk 2 above)
Comment (lproven) [reply xml/admin_integration.xml]: Agreed. Fixed -- I have also moved the link text outside the tag, which is more consistent with the other Guides.

I did bring entities for SES, MicroOS and a few other things across from the release-notes entities file.
